kotlinx-coroutines-core/common/src/Job.kt

```diff
@@ -42,22 +47,30 @@ import kotlin.jvm.*
  * | _Cancelled_ (final state)        | `false`    | `true`        | `true`        |
  * | _Completed_ (final state)        | `false`    | `true`        | `false`       |
  *
- * Usually, a job is created in the _active_ state (it is created and started). However, coroutine builders
+ *
+ * Note that these states are mentioned in italics below to make them easier to distinguish.
+ *
+ * Usually, a job is created in the _active_ state (it is created and started).
+ * However, coroutine builders
  * that provide an optional `start` parameter create a coroutine in the _new_ state when this parameter is set to
- * [CoroutineStart.LAZY]. Such a job can be made _active_ by invoking [start] or [join].
+ * [CoroutineStart.LAZY].
+ * Such a job can be made _active_ by invoking [start] or [join].
  *
- * A job is _active_ while the coroutine is working or until [CompletableJob] is completed,
- * or until it fails or cancelled.
+ * A job is in the _active_ state while the coroutine is working until the [CompletableJob] completes,
```

Comment:
The original "or" was correct: either the coroutine corresponding to the job completes, or `CompletableJob.complete` is called.